src/main/java/com/rapid7/client/dcerpc/mslsad/LocalSecurityAuthorityService.java

```diff
@@ -38,20 +46,52 @@
  * @see <a href= "https://msdn.microsoft.com/en-us/library/cc234420.aspx">[MS-LSAT]</a>
  */
 public class LocalSecurityAuthorityService {
+    private ContextHandle policyHandle;
```

Comment:
If the policy handle is persisted, what happens when it is later closed or becomes invalid?

Reply:
@bli-r7 , good point! .   As I mentioned the above, it's just testing code for share people how to use the policyHanldle for example( because we don't know exactly  who open the policyHandle  or reuse it or close (currently the implementation is not fixed so later we should refactor  based on usage scenario ).  It's my thinking :)

Reply:
I actually think the best use case would be for the client to pass a ContextHandle to each method; this would allow them to control the lifecycle. Alternatively, you could open/close a ContextHandle on each request. Persisting it in the class would likely result in an invalid ContextHandle at some point.